### PR TITLE
Feat/dynamic search filter

### PR DESCRIPTION
**Pull request by Supakrit65**
image
image

### CODE REVIEW

File: src/pages/sports.tsx
Comment (Savetang19) [inline src/pages/sports.tsx:1]: Can you add `Spin` icon as loading status to this page? It's take a couple of seconds show blank page before show all of the sport cards.

**Review by Savetang19 — CHANGES_REQUESTED** [review]

**Review by Savetang19 — APPROVED** [review]
image

Me approved

**Review by RyukungG — APPROVED** [review]

**Review by MarojT — APPROVED** [review]
It work well.

**Pull request merged**